source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -233,6 +233,11 @@ - (void)didFetchHttpRequest:(NSURLRequest *)request
     NSLog(@"Http Request fetched: %@", request);    
 }
 
+- (void)didLoadElements
```

Comment:
Renamed to didLoadElements to better represent its functionality. The previous name didCompleteRendering sounded overpromising.